### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by thedekerone**
In this pull request i finished building all the enumerable methods

### CODE REVIEW

**Comment by pelzolga123** [comment]
### Hi Mauricio,
I am marking this Pull Request as invalid due to the following reason:
* Wrong Git Flow. Correct Git Flow: production `master` branch + main `development` branch + Pull Requests from feature branches to the `develop` ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/git_flow.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/57421589/))
So basically you need to have 3 branches: `master`, `development`, and `feature`. Close this Pull Request and create new one from the `feature` branch to `development`.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*
